src/ucp/api/ucp.h

```diff
@@ -213,6 +213,24 @@ enum ucp_ep_params_flags_field {
 
 
 /**
+ * @ingroup UCP_ENDPOINT
+ * @brief @anchor ucp_ep_close_flags Close UCP endpoint flags.
+ *
+ * The enumeration allows specifying behavior of @ref ucp_ep_close routine.
+ */
+enum {
+    UCP_EP_CLOSE_FLAG_ONE_SIDED          = 0, /**< @ref ucp_ep_close releases
+                                                   the endpoint without any
+                                                   confirmation from the peer.
+                                                   All outstanding requests will
+                                                   be completed with
+                                                   @ref UCS_ERR_CANCELED error. */
+    UCP_EP_CLOSE_FLAG_FLUSH              = 1, /**< @ref ucp_ep_close flushes all
```

Comment:
should use UCS_BIT() since these are flags
Do we really need to add flag for one-sided close? this can be the default.. Or maybe rename it to UCP_EP_CLOSE_FOCE or UCP_EP_CLOSE_TEARDOWN

Reply:
> should use UCS_BIT() since these are flags

but they can't be or'ed...

> this can be the default.

Yes, this is why it's '0' :)

Reply:
so they should not be called "flags"...

Reply:
"mode" or "type"?

Reply:
mode